tests/contenttypes_tests/tests.py

```diff
@@ -440,6 +443,52 @@ def test_interactive_false(self):
         self.assertEqual(ContentType.objects.count(), self.before_count + 1)
 
 
+class RemoveContentTypesTests(TestCase):
+    def setUp(self):
+        self.before_count = ContentType.objects.count()
+        ContentType.objects.create(app_label='contenttypes_tests', model='Fake')
+        self.app_config = apps.get_app_config('contenttypes_tests')
+
+    def test_remove_contenttypes(self):
+        """
+        remove_contenttypes should call update_contenttypes.
+        """
+        with mock.patch('django.contrib.contenttypes.management.update_contenttypes') as uct:
+            remove_contenttypes(self.app_config)
+            uct.assert_called
```

Comment:
Need to test how many times `update_contenttypes` was called and test arguments which passed to it.